### PR TITLE
Feature/hide indicators

### PR DESCRIPTION
**Pull request by tet3**
Started messing around just trying to add functionality for #5 but got carried away and added the SLDS icon option. Though I think a new namespaced pull of SLDS is required to get some newer icons - there are several new ones in the Standard set, and Custom 101-113 are also not in the current static resource.

I addressed a couple of cases with #6 -- added the PL values, and explicitly cast to String with valueOf() and then to Decimal. It would be great to handle dates, though.

### CODE REVIEW

File: src/classes/Indicators_Service.cls [hunk 1]
```diff
@@ -1,67 +1,117 @@
...
+	}
 
-        public Indicator(String value, String color, Boolean display, String hover) {
-            this.Value = value; this.Color = color; this.Display = display; this.Hover = hover;
-        }
-    }
+	public static Set<String> spriteSet {
+		get {
+			if (spriteSet == null) {
+/*				if (Cache.Org.contains('indicatorbar.partition1.spriteSet')) {
```
Comment (tet3) [inline src/classes/Indicators_Service.cls]: I thought Platform Cache would be a cool way to make the list of icon names be a little quicker, but a) I had to request it in my dev org, and b) the managed package Platfom Cache scenario is sub-optimal for OSS projects anyway. From https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_cache_partition_setup.htm: 
> If you’re working with managed packages, we recommend using Branch Packaging Orgs to share a namespace across partitions. This feature lets you maintain multiple orgs or partitions as “branches” of your primary org. For information about Branch Packaging Orgs, contact Salesforce.

File: src/classes/Indicators_Service.cls [hunk 2]
```diff
@@ -1,67 +1,118 @@
...
+					Set<String> idNames = new Set<String>();
+
+					if (idMatcher.find()) {
+					    do {
+					        idNames.add(idMatcher.group().substringAfter('="'));
+					    } while (idMatcher.find());
+					}
+					// see above re: org cache
+					// Cache.Org.put('indicatorbar.partition1.spriteSet',idNames);
+					return idNames;
```
Comment (cdcarter) [inline src/classes/Indicators_Service.cls]: you'd need to assign the value to the spriteSet member for it to actually be memoized.

File: cumulusci.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+project:
+    name: Weathervane
+    package:
+        name: Weathervane
+        namespace: indicatorbar
```
Comment (cdcarter) [inline cumulusci.yml]: include an explicit api version here - since the valueset name change in the .object is breaking-y

**Review by cdcarter — CHANGES_REQUESTED** [review]
thomas - I will take a look  at this in an org tomorrow but on first glance it looks good, a few comments in this review.

though..., making a callout just for runtime validation of the sprite name seems like a bad idea (especially when we're doing things like casting through string and decimal adhoc.) since we're bundling a specific slds resource, you can simply inline the list in the class as a final constant instead.

File: cumulusci.yml (hunk 3 above)
Comment (tet3) [reply cumulusci.yml]: @cdcarter 38.0 is ok, right? It's 39.0 that you're wanting to avoid?

Comment (cdcarter) [reply cumulusci.yml]: yep!

**Review by cdcarter — APPROVED** [review]
Looks awesome!

**Pull request merged**